get_access_token.sh

```diff
@@ -12,13 +12,27 @@ else
 fi
 
 source ${DIR_NAME}/kas-installer.env
+source ${DIR_NAME}/kas-fleet-manager/kas-fleet-manager-deploy.env
 
 GRANT_TYPE=''
 USER_PARAMS=''
 KEYCLOAK_REALM=${KEYCLOAK_REALM:-rhoas}
 
 case ${1} in
     "--owner" )
+        if [ "${RHOAS_TOKEN:-false}" = 'true' ] ; then
+            RHOAS_WHOAMI=$(rhoas whoami)
+
+            if [ ${?} -ne 0 ]; then
+                echo ${RHOAS_WHOAMI} > /dev/stderr
+                exit 1
+            fi
+
+            ACCESS_TOKEN=$(cat ${HOME}/.config/rhoas/config.json | jq -r '.access_token')
+            printf "%s\n" ${ACCESS_TOKEN}
+            exit 0
+        fi
```

Comment:
Currently no other way to obtain the "owner" token from sso.qa.redhat.com. We'll need a way to accomplish this without user interaction.